src/security/TokenEncoding.ts

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Supported token encodings.
+ */
+export enum TokenEncoding {
+    ASCII = 'ASCII',
+    BASE64 = 'BASE64',
+}
```

Comment:
Should the values be in lower case?

Reply:
Also, should `BASE64` replace `"base64"` constants?

Reply:
I didn't get the second comment. Could you please explain it again?

Reply:
For all examples/tests etc. we use `"base64"`, `"ascii"` etc. Wouldn't it better to use `TokenEncoding.ASCII`, etc. instead of them?

Reply:
Also, in the documentation...

Reply:
Oh, I see. I personally like the string constant usage, as it is much easier, but as you suggested, I have changed the usages to enum members.